### DISCUSSION TITLE
Getting duals from SCIP

### DISCUSSION THREAD

**Discussion by ianluik** [google/or-tools#3716] [Linear Solver questions] [answered]
Hi,

I tried getting duals from a linear model with SCIP as back-end, but I get the error "Dual value only available for continuous problems". If I use gurobi as solver I get the same problem, but not if I use gurobi_cl.

My question is: is there any way to get the duals from SCIP through OR-tools? I did not find a specific lp-version of SCIP in the supported solvers.

Thanks in advance!

**Comment by lperron** [comment, marked as answer]
No, SCIP is MIP only.
Use GLOP.
